### PR TITLE
Add PR template

### LINKED ISSUE #6: Add PR Template
Create `PULL_REQUEST_TEMPLATE.md` file in root directory with the following content

```
Fixes #issuenumber
-

Check everything which applies

- [ ] I have added the issue number for which this pull request is created.

-

### PR DESCRIPTION
**Pull request by zswen**
Fixes #6 
-
Hello,
This is my first pull request. Please let me know if I did it correctly. 
Thank you very much

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Fixes #issuenumber
+-
+
+Check everything which applies
+
+- [ ] I have added the issue number for which this pull request is created.
+
+-
```
Comment (arshadkazmi42) [inline PULL_REQUEST_TEMPLATE.md]: can you remove this `-`

Comment (zswen) [reply PULL_REQUEST_TEMPLATE.md]: Sure no problem!

Comment (zswen) [reply PULL_REQUEST_TEMPLATE.md]: Got it updated. Thank you very much for the review @arshadkazmi42 !

Comment (arshadkazmi42) [reply PULL_REQUEST_TEMPLATE.md]: You need to remove the `-` at line 8
At line 6 we want the `-`

**Pull request merged**